containers/codespaces-linux/history/dev.md

```diff
@@ -113,15 +101,14 @@ mcr.microsoft.com/vscode/devcontainers/universal:dev
 | [yarn](https://yarnpkg.com/) | 1.22.15 | /opt/yarn |
 | [Maven](https://maven.apache.org/) | 3.6.3 | /opt/maven |
 | [Gradle](https://gradle.org/) | 7.4.2 | 
-| Docker (Moby) CLI &amp; Engine | 20.10.14+azure | 
+| Docker (Moby) CLI &amp; Engine | 20.10.14 | 
```

Comment:
This file is auto-generated (so nbd), but this `+azure` is metadata showing that we installed moby from the microsoft azure repo (not related to the azure cli, which I suspect is why this was removed)

Reply:
ah! my bad. I'll revert this file changes 👍